TASK-4/CandidateReportGenerator.cs

```diff
@@ -0,0 +1,33 @@
+namespace Persons;
+
+public class CandidateReportGenerator : IReportGenerator
+{
+    private IEnumerable<Candidate> _listOfCandidates;
+
+    public CandidateReportGenerator(IEnumerable<Candidate> listOfCandidates)
+    {
+        _listOfCandidates = listOfCandidates;
+    }
+
+    public void Report()
+    {
+        const string template = "{0,36} | {1,20} | {2, -40} | {3,25} | {4, 18}";
+
+        Console.WriteLine(template,
+            "Id:", "Name Surname:", "Desired position:", "Position description:", "Desired salary:");
+        
+        for (int i = 0; i < _listOfCandidates.Count(); i++)
```

Comment:
Here and for employee it would be better to use foreach

Reply:
Done.